projects/opendr_ws/src/perception/scripts/object_tracking_2d_deep_sort.py

```diff
@@ -174,36 +184,62 @@ def draw_predictions(frame, predictions: TrackingAnnotation, is_centered=False,
         )
 
 
-if __name__ == "__main__":
-    # Automatically run on GPU/CPU
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
-
-    # initialize ROS node
-    rospy.init_node("opendr_deep_sort", anonymous=True)
-    rospy.loginfo("Deep Sort node started")
-
-    model_name = rospy.get_param("~model_name", "deep_sort")
-    temp_dir = rospy.get_param("~temp_dir", "temp")
-    input_image_topic = rospy.get_param(
-        "~input_image_topic", "/opendr/dataset_image"
-    )
-    rospy.loginfo("Using model_name: {}".format(model_name))
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--model_name", help="Name of the trained model",
+                        type=str, default="deep_sort")
```

Comment:
Added choices argument, it seems redundant but it shows up when calling help and avoids users providing wrong values.

```suggestion
    parser.add_argument("-n", "--model_name", help="Name of the trained model",
                        type=str, default="deep_sort", choices=["deep_sort"])
```